Lecture4/webservices/webservices/NotesService.java

```diff
@@ -0,0 +1,75 @@
+package webservices;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.util.EntityUtils;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class NotesService {
```

Comment:
Could you please pass a so called HttpMethods and ObjectMapper instances from the outside?